Shawntru/Rotten-Tomatillos
# Feature/api calls

## Is this a fix or a feature?
Feature
## What is the change?
Reorganize file structure, create API fetches for homepage and individual movies, created error catch with error display page
## Where should the reviewer start?
App.js line 2
## How should this be tested?
Ensure page loads correctly, api calls are performed correctly, and verify error catching
## Relevant Ticket(s)
#8 #9 #11 #12 #13
